src/lib/punctuation/double-quotes.js

```diff
@@ -26,56 +26,92 @@
 	@returns {string} output with properly replaces double qoutes and double primes
 */
 export function fixDoubleQuotesAndPrimes(string, locale) {
-
 	/* [0] Remove extra terminal punctuation around double quotes
 					e.g. “We will continue tomorrow.”.
 
 					Exception
 					Do not remove extra terminal punctuation for the given example:
 					Byl to “Karel IV.”, ktery neco
 					*/
-	let pattern = "([^" + locale.romanNumerals + "])([" + locale.sentencePunctuation + "])("+ locale.doubleQuoteAdepts + ")([" + locale.sentencePunctuation + "])";
-	let re = new RegExp(pattern, "g");
-	string = string.replace(re, "$1$2$3");
+	let pattern =
+		'([^' +
+		locale.romanNumerals +
+		'])([' +
+		locale.sentencePunctuation +
+		'])(' +
+		locale.doubleQuoteAdepts +
+		')([' +
+		locale.sentencePunctuation +
+		'])'
+	let re = new RegExp(pattern, 'g')
+	string = string.replace(re, '$1$2$3')
 
 	/* [1] Swap right double quote adepts with a terminal punctuation */
 	pattern =
-					"("+ locale.doubleQuoteAdepts + ")"
-				+ "([" + locale.terminalPunctuation + locale.ellipsis + "])";
-	re = new RegExp(pattern, "g");
-	string = string.replace(re, '$2$1');
+		'(' +
+		locale.doubleQuoteAdepts +
+		')' +
+		'([' +
+		locale.terminalPunctuation +
+		locale.ellipsis +
+		'])'
+	re = new RegExp(pattern, 'g')
+	string = string.replace(re, '$2$1')
 
 	/* [2] Identify inches, arcseconds, seconds
 				 Note: we’re not using locale.doubleQuoteAdepts variable
 				 as commas and low-positioned quotes are ommited*/
-	pattern = "(\\d[" + locale.spaces +"]?)(“|”|\"|″|‘{2,}|’{2,}|'{2,}|′{2,})",
-	re = new RegExp(pattern, "g");
-	string = string.replace(re, "$1{{typopo__double-prime}}");
+	;(pattern =
+		'(\\d[' + locale.spaces + ']?)(“|”|"|″|‘{2,}|’{2,}|\'{2,}|′{2,})'),
+		(re = new RegExp(pattern, 'g'))
+	string = string.replace(re, '$1{{typopo__double-prime}}')
```

Comment:
I'm not sure I follow this optimization :)

Reply:
There was a `,` at the end of line 51 that baffled the parser.